docs/esa/jwst/jwst.rst

```diff
@@ -326,6 +326,15 @@ Using the observation ID as input parameter, this function will retrieve the obs
    'jw02739001001_02105_00002_nrcblong',
    'jw02739001001_02105_00003_nrcalong']
 
+To query the data products associated with a certain Proposal ID and filtered by product_type.
+
+.. doctest-remote-data::
+
+  >>> from astroquery.esa.jwst import Jwst
+  >>> observation_list = Jwst.download_files_from_program(proposal_id='6651', product_type='preview')
+  >>> print(observation_list) # doctest: +IGNORE_OUTPUT
+  [np.str_('jw06651001001_05201_00001_nis'), np.str_('jw06651002001_05201_00001_nis')]
```

Comment:
Interesting that the values come up wrapped in a `np.str_` as opposed to be just strings, and thus leading to this somewhat ugly print as opposed to (of course looping through the values works nicely, but that would be uglier for the docs)
`['jw06651001001_05201_00001_nis', 'jw06651002001_05201_00001_nis']`


Maybe this can be have a look into sometimes in the future in `JwstClass.get_decoded_string`? 

And here, given that the output is already been ignored, I would suggest to format it nicely.


```suggestion
  ['jw06651001001_05201_00001_nis', 'jw06651002001_05201_00001_nis']
```